requirements/local-docs-build.txt

```diff
@@ -1,7 +1,7 @@
 -r pip.txt
 
 # Base packages
-docutils==0.14
+docutils==0.15
```

Comment:
@humitos 
Tests are failing in [readthedocs-sphinx-ext](https://github.com/readthedocs/readthedocs-sphinx-ext) due to this update.